chef/cookbooks/ceph/recipes/mds.rb

```diff
@@ -0,0 +1,47 @@
+if node.roles.include?("ceph-mon")
+  include_recipe "ceph::default"
+  include_recipe "ceph::conf"
+else
+  include_recipe "ceph::keyring"
+end
+include_recipe "ceph::server"
+
+execute "create mds keyring" do
+  command "ceph auth get-or-create mds.#{node["hostname"]} osd 'allow rwx' mds 'allow' mon 'allow profile mds' -o /etc/ceph/mds.#{node["hostname"]}.keyring ; chown ceph.ceph /etc/ceph/mds.#{node["hostname"]}.keyring"
```

Comment:
Should it not say `mds 'allow *` instead of `mds 'allow'`? I see that the ceph doc has an example like this, but in other places the star is explicitly listed.